Dockerfile

```diff
@@ -20,7 +20,7 @@ ENV GEM_HOME $WORK_ROOT/bundle
 ENV BUNDLE_BIN $GEM_HOME/gems/bin
 ENV PATH $GEM_HOME/bin:$BUNDLE_BIN:$PATH
 
-RUN gem install bundler
```

Comment:
@grosendo2006 This change shouldn't be necessary, cause here:
https://github.com/rootstrap/rails_api_base/blob/f267f15d1aa2b9ca16a7e71937ae580e38134156/Dockerfile#L33
we're already upgrading the bundler version to the latest. WDYT?